app/models/role.rb

```diff
@@ -57,6 +57,13 @@ def roleid_from_username(account, login)
       tokens.join(":")
     end
 
+    def roleid_from_host(account, login)
```

Comment:
i don't think that you need this method. the method `roleid_from_username` works for both hosts and users. am i missing something?

Reply:
Before i done this policies like this didn't work
![image](https://user-images.githubusercontent.com/73157235/119309425-f6572080-bc76-11eb-9df0-58b37f64e6d8.png)
Because the role it created was account:host:role_name but it searched for account:user:role_name
Maybe the way i loaded the policy is wrong?

Reply:
did you pass the request using `host/bla` or just `bla`? hosts should be prefixed using `host/`

Reply:
now you can better understand the logic of `roleid_from_username`